main.py

```diff
@@ -99,33 +104,49 @@ def get_closed_issues(
 ) -> List[Tuple[FileIssue, github.Issue.Issue]]:
     closed_issues = []
     for repo, issues in issues_by_repo(issues).items():
-        gh_repo = gh.get_repo(repo)
-        for issue in issues:
-            try:
-                gh_issue = gh_repo.get_issue(number=issue.num)
-            except github.GithubException.UnknownObjectException:
-                logging.error("failed to look up issue %r", issue.ref)
-            else:
-                if gh_issue.state == "closed":
-                    closed_issues.append((issue, gh_issue))
+        try:
+            gh_repo = gh.get_repo(repo)
+        except UnknownObjectException:
+            logging.error("failed to look up issue %r", issue.ref)
+        else:
+            for issue in issues:
+                try:
+                    gh_issue = gh_repo.get_issue(number=issue.num)
+                except github.GithubException.UnknownObjectException:
+                    logging.error("failed to look up issue %r", issue.ref)
+                else:
+                    if gh_issue.state == "closed" and gh_issue not in [ghi[1] for ghi in closed_issues]:
+                        closed_issues.append((issue, gh_issue))
     return closed_issues
 
 
+def get_unique_issues(
+    issues: List[FileIssue]
+) -> Dict[str, List[FileIssue]]:
+    """Return all the name and line number of all the files that contain the issue"""
+    dedupe_issues = {}
+    for issue in issues:
+        for ref in dedupe_issues:
+            if ref == issue.ref:
+                dedupe_issues[ref].append(issue)
+                break
+        else:
+            dedupe_issues[issue.ref] = [issue]
+    return dedupe_issues
+
+
 def get_issue_locations(
-    issues: List[FileIssue], issue: FileIssue
+    unique_issues: Dict[str, List[FileIssue]], closed_issue: FileIssue
```

Comment:
```suggestion
    issues: Dict[str, List[FileIssue]], issue: FileIssue
```

can revert these changes